sg_otio/sg_settings.py

```diff
@@ -347,22 +348,58 @@ def shot_cut_fields_prefix(self, value):
         self._shot_cut_fields_prefix = value
 
     @property
-    def shot_omitted_statuses(self):
+    def shot_omit_status(self):
         """
-        Return the list of omitted statuses used for Shots.
+        Return the SG Shot status to use when omitting Shots.
+
+        :returns: A SG status short code or ``None``.
+        """
+        return self._shot_omit_status
+
+    @shot_omit_status.setter
+    def shot_omit_status(self, value):
+        """
+        Set the SG Shot status to use when omitting Shots.
+
+        :param value: A SG status short code or ``None``.
+        """
+        self._shot_omit_status = value or None
+
+    @property
+    def shot_reinstate_status(self):
+        """
+        Return the SG Shot status to use when reinstating Shots.
+
+        :returns: A SG status short code or ``None``.
+        """
+        return self._shot_reinstate_status
+
+    @shot_reinstate_status.setter
+    def shot_reinstate_status(self, value):
+        """
+        Set the SG Shot status to use when omitting Shots.
```

Comment:
maybe add something about the magic constant "Previous Status" and how it works?